tutorials/Land/Soil/Water/equilibrium_test.jl

```diff
@@ -45,24 +36,24 @@ using ClimateMachine.BalanceLaws:
 
 
 # - define the float type desired (`Float64` or `Float32`)
-FT = Float64;
+const FT = Float64;
```

Comment:
Should we check if this tutorial finishes in a reasonable time with `const FT`? Maybe we should add it to the tutorials pages array ([here](https://github.com/CliMA/ClimateMachine.jl/blob/master/docs/list_of_tutorials.jl#L21-L50))

Reply:
ill check the time before and after making the change

Reply:
hey so we ran it with and without the constant, and it took about twice as long with the const, so we left it as is for now but can look into this in next pr